pkg/resources/object_renaming_lists_and_sets_acceptance_test.go

```diff
@@ -0,0 +1,463 @@
+package resources_test
+
+import (
+	"fmt"
+	"testing"
+
+	acc "github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/bettertestspoc/assert"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/acceptance/testenvs"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/resources"
+	"github.com/Snowflake-Labs/terraform-provider-snowflake/pkg/sdk"
+	"github.com/hashicorp/terraform-plugin-testing/helper/resource"
+	"github.com/hashicorp/terraform-plugin-testing/tfversion"
+)
+
+func TestAcc_BasicListFlow(t *testing.T) {
```

Comment:
General thought: this test is pretty straightforward; it just checks if the state is equal to config but does not validate suppressing diff/actions that happen underneath, so I am not sure if it really proves something. Let's discuss this today.

Reply:
Maybe if we keep access to "external state" we could also have access to some resource actions log, and we could verify what actions are happening (so that we can better validate if the logic conforms with the expectations and so that we can later use it in practice)